networkit/cpp/io/test/IOGTest.cpp

```diff
@@ -826,6 +828,8 @@ TEST_F(IOGTest, testNetworkitBinaryKonect) {
 
 	NetworkitBinaryReader reader;
 	Graph G2 = reader.read("output/binary_konect");
+	EXPECT_EQ(G2.isDirected(), true);
+	EXPECT_EQ(G2.isWeighted(), true);
 	ASSERT_EQ(G2.numberOfEdges(), G.numberOfEdges());
 	ASSERT_EQ(G2.numberOfNodes(), G.numberOfNodes());
 	G.forNodes([&](node u){
```

Comment:
IMHO tests should not only check whether the edges exist, but also whether their weights match.